doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -0,0 +1,91 @@
+"""
+============================================================
+Reconstruct dust-covered human cornea image using inpainting
```

Comment:
```suggestion
Reconstruct dust-obstructed human cornea image with inpainting
```
Clearer title as per @lagru 's suggestion.